includes/class-wc-gateway-ppec-plugin.php

```diff
@@ -161,8 +161,13 @@ public function bootstrap() {
 			$this->_check_credentials();
 
 			$this->_bootstrapped = true;
-			delete_option( 'wc_gateway_ppce_bootstrap_warning_message' );
-			delete_option( 'wc_gateway_ppce_prompt_to_connect' );
+            if ( false !== get_option( 'wc_gateway_ppce_bootstrap_warning_message' ) ) {
```

Comment:
Please use tabs instead of spaces and wrap these in a `is_admin()` check as @WPprodigy mentioned.

Reply:
Oops, looks like my editor wasn't configured quite right. Have rectified and opened this PR to add an [editorconfig](https://editorconfig.org/) file https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/pull/543